extensions/ql-vscode/src/view/model-editor/MethodRow.tsx

```diff
@@ -47,6 +49,14 @@ const ApiOrMethodRow = styled.div`
   gap: 0.5em;
 `;
 
+const ButtonsContainer = styled.div`
```

Comment:
Should this be a bit more specific e.g. `ModelButtonsContainer`? We have other buttons (e.g. Usages button)

Reply:
Good point, done!